src/atomate2/vasp/jobs/lobster.py

```diff
@@ -61,14 +61,14 @@ class LobsterStaticMaker(BaseVaspMaker):
     name: str = "static_run"
     input_set_generator: VaspInputGenerator = field(
         default_factory=lambda: StaticSetGenerator(
-            user_kpoints_settings={"grid_density": 6000},
+            user_kpoints_settings={"reciprocal_density": 400},
             user_incar_settings={
                 "IBRION": 2,
```

Comment:
And IBRION, ISIF, NSW from here.

Reply:
Short question: should I enable auto_ispin and things like this here as well?

Reply:
Yeah, I think you can enable it. The default is for the lobster workflow to do a relaxation anyway, so I think it could be useful.